lib/players/human.rb

```diff
@@ -0,0 +1,8 @@
+class Player::Human < Player
+
+  def move(location = "")
+    location = gets.chomp
+    location
+  end
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(location = "")
  gets.chomp
end
```